constants.js

```diff
@@ -40,4 +40,21 @@ export const tokensHost = `https://${chainName}all-tokens.webaverse.com`;
 export const landHost = `https://${chainName}sidechain-land.webaverse.com`;
 export const web3MainnetSidechainEndpoint = 'https://mainnetsidechain.exokit.org';
 export const web3TestnetSidechainEndpoint = 'https://testnetsidechain.exokit.org';
-export const homeScnUrl = `https://webaverse.github.io/street/street.scn`;
+export const homeScnUrl = 'https://webaverse.github.io/street/street.scn';
+
+// Setup dialog URL, use hash if on githubpreview and its an okay hash
+const rGithub = /\.githubpreview\.dev$/;
+const isGH = rGithub.test(window.location.hostname);
+
+const hashHost =
+  /^[a-z][a-z0-9+\-.]*:\/\/([a-z0-9\-._~%!$&'()*+,;=]+@)?([a-z0-9\-._~%]+|\[[a-z0-9\-._~%!$&'()*+,;=:]+\])/.exec(
+    window.location.hash.slice(1),
+  );
+
+const isOKHASH = hashHost && hashHost[2] && rGithub.test(hashHost[2]);
+
+export const dialogUrl = isGH
```

Comment:
Would prefer it as

```
isGH ?
isOKHASH
: 
'dialog.webaverse.com';
```

I don't know how to get the linter to do that yet, but since this is currently inconsistent with the codebase it's a little hard to read.

Reply:
![image](https://user-images.githubusercontent.com/462380/130139235-f73563c8-3299-4cc4-b17f-25ce9f4a551d.png)
Is this correct?